Contentstack.Core/ContentstackClient.cs

```diff
@@ -34,6 +41,8 @@ internal string StackApiKey
             set;
         }
         private ContentstackOptions _options;
+        private HttpClient _httpClient;
```

Comment:
@AnkurContentstack HTTPClient taken from Management SDK We already have different implementation for Delivery SDK.